utilities/transactions/write_prepared_txn.cc

```diff
@@ -46,13 +46,15 @@ void WritePreparedTxn::MultiGet(const ReadOptions& options,
                                 PinnableSlice* values, Status* statuses,
                                 bool sorted_input) {
   SequenceNumber min_uncommitted, snap_seq;
-  const bool backed_by_snapshot =
+  const SnapshotBackup backed_by_snapshot =
```

Comment:
How about WriteUnprepared? Does not it use AssignMinMaxSeqs or ValidateSnapshot?

Reply:
We'll probably need a wup version of `AssignMinMaxSeqs`. `WritePreparedTxnDB::ValidateSnapshot` can just take the wp version, because I don't think it needs to access anything specific to `WriteUnpreparedTxnReadCallback`.

I don't feel too strongly about this. I was just wondering if you think it's cleaner.